engine/lib/output.php

```diff
@@ -153,6 +153,12 @@ function elgg_format_attributes(array $attrs = array()) {
 
 		unset($attrs['js']);
 	}
+	
+	if (isset($attrs['data-colorbox-opts'])) {
+		if (!@json_decode($attrs['data-colorbox-opts'])) {
```

Comment:
Seems like we could detect this problem client-side by sniffing the raw attribute content. I just really don't want this function to start taking on validation of business logic.